public/javascripts/ViewCarts.js

```diff
@@ -344,7 +345,15 @@ function handleDirtyItemsRecursively(entry, cartItem) {
             handleDirtyItemsRecursively(nextEntry, cartItem);
         })
         .fail(function (res) {
-            $("#response").text("Error: handleDirtyItemsRecursively: Connection error.");
```

Comment:
good catch, i didn't change it because there is a .then call afterword, but i needed to add window.apiroute to the front of the getcall